client/my-sites/plans/current-plan/style.scss

```diff
@@ -148,3 +146,136 @@
 	margin-bottom: 10px;
 	text-align: center;
 }
+
+body.is-section-plans.is-trial-plan {
+	&.theme-default.color-scheme {
+		--color-surface-backdrop: #fff;
+	}
+
+	.current-plan__content {
+		.section-nav {
+			box-shadow: inset 0 -1px 0 rgba(0, 0, 0, 0.05);
+		}
+
+		.current-plan__trial-card {
+			display: flex;
+			padding: 30px;
+			background-color: #f6f7f799;
+			box-shadow: none;
+
+			.current-plan__trial-card-content {
+				flex: 1;
+				padding-right: 50px;
+				.badge {
+					border-radius: 0;
+					background-color: #000;
+					color: #fff;
+					font-size: 0.75rem;
+				}
+				.current-plan__card-title {
+					font-size: $font-title-medium;
+					font-weight: 400;
+					color: #2f2f2f;
+					margin: 15px 0;
+				}
+				.current-plan__card-subtitle {
+					font-size: 1rem;
+					font-weight: 400;
+					color: #757575;
+				}
+			}
+		}
+
+		.current-plan__section-title {
+			font-size: $font-title-medium;
+			text-align: center;
+			font-weight: 400;
+			color: #2f2f2f;
+			margin: 40px 0 10px 0;
+		}
+		.current-plan__section-subtitle {
+			text-align: center;
+			font-size: 1rem;
+			color: #757575;
+		}
+
+		.current-plan__included-wrapper {
+			display: flex;
+			flex-flow: row wrap;
+			list-style: none;
+			padding: 0;
+			margin-top: 30px;
+
+			.card {
+				width: 240px;
+				display: flex;
+				flex-direction: column;
+				border: 1px solid #dcdcde;
+				border-radius: 2px;
+				padding: 48px 24px 35px 24px;
+				box-shadow: none;
+
+				.current-plan__included-ilustration {
+					width: 60px;
+				}
+
+				.current-plan__included-title {
+					margin-top: 20px;
+					font-weight: 600;
+					font-size: 1rem;
+				}
+				.current-plan__included-text {
+					flex: 1;
+					font-weight: 400;
+					font-size: 0.875rem;
+					color: #757575;
+				}
+				.current-plan__included-link {
+					color: #c9356e;
+					font-size: 0.875rem;
+					border: 0;
+					padding: 0;
+					text-align: left;
+				}
+			}
+		}
+
+		.current-plan__more-wrapper {
+			display: flex;
+			flex-flow: row wrap;
+			list-style: none;
+			padding: 20px;
+			margin-top: 30px;
+
+			.card {
+				width: 490px;
+				display: flex;
+				box-shadow: none;
+
+				.current-plan__more-ilustration {
+					width: 130px;
+					height: fit-content;
+				}
+				.current-plan__more-content {
+					padding-left: 25px;
+
+					.current-plan__more-title {
+						font-weight: 600;
+						font-size: 1rem;
+						margin-bottom: 1rem;
+					}
+					.current-plan__more-text {
+						font-weight: 400;
+						font-size: 0.875rem;
+						color: #757575;
+					}
+				}
+			}
+		}
+
+		.current-plan__cta-wrapper {
+			text-align: center;
+			margin-top: 30px;
+		}
+	}
+}
```

Comment:
Can you check for a few items in the CSS?
1. Do we have these colours available as variables? If so, we should use the variables rather than hard-coded colours. If we don't have them yet, we should consider creating variables (even if local) to add them, especially because there is re-use within the current file.
2. Have you considered/explored the layout on mobile or tablet screens? Some of these hard-coded widths and margins feel risky to me without that.